### ISSUE TITLE
atmospheric model

### ISSUE THREAD

**Issue by thomasgas** [cta-observatory/ctapipe-extra#24]
the atmospheric model that we have now is just the one for paranal (called atmprof26 in corsika&simtelarray).  
We should add also the model for La Palma. Is it a problem if I take the atmospheric model for La Palma and I add it to this repository with the same format used for paranal?

**Comment by kosack** [comment]
no that should be ok for now.  In the future, there will be different service data for each site, but until that system is in place, we can just put a few here.  Of course ctapipe-extra was just supposed to be defaults (not all calibration and configuration data possible), and the users can specify their own files by adding directories to $CTAPIPE_SVC_PATH, but perhaps that could lead to confusion?

**Comment by maxnoe** [comment]
The new simtel monte carlos include the atmospheric profile used for the simulation

**Comment by thomasgas** [comment]
what does "new" means? like prod4?
the header will just have the name of the atmospheric model or also the model itself? 
thanks!

**Comment by maxnoe** [comment]
There is a new eventio type (1216) that contains the full atmprof data. See https://github.com/cta-observatory/pyeventio/pull/188

I think this is a matter of the version of the `iact/atmo` extension. 1.60 I think.

**Comment by maxnoe** [comment]
Yes: https://www.mpi-hd.mpg.de/hfm/~bernlohr/iact-atmo/ChangeLog

**Comment by kosack** [comment]
we still need an atmospheric model when analyzing real data, so perhaps we 
1. switch to reading the atmosphere model from MCs  in SimTelEventSource
2. improve `ctapipe-dump-instrument` to also write out the model in a nice table format (FITS or ECSV, using `astropy.table`)
3. include any useful models as tables here, so they can be loaded later for computing h_max, etc.  

Soon, `ctapipe-extra` will be deprecated and replaced with just a file store (already partially working), an we can manage multiple sub-configurations (e.g. `prod3/*`, `prod4/*`), in which there are subarray descriptions, atmosphere profiles, etc.
